### ISSUE TITLE
Crash due to ACCESS_WIFI_PERMISSION

### ISSUE THREAD

**Issue by Pavel87** [socure-inc/socure-sigmadevice-sdk-android#14]
Can you document here somewhere that this permission is required?

```
java.lang.SecurityException: WifiService: Neither user 10354 nor current process has android.permission.ACCESS_WIFI_STATE.
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3011)
	at android.os.Parcel.createException(Parcel.java:2995)
	at android.os.Parcel.readException(Parcel.java:2978)
	at android.os.Parcel.readException(Parcel.java:2920)
	at android.net.wifi.IWifiManager$Stub$Proxy.getConnectionInfo(IWifiManager.java:2836)
	at android.net.wifi.WifiManager.getConnectionInfo(WifiManager.java:3829)
	at com.socure.idplus.devicerisk.androidsdk.viewModel.network.NetworkViewModel.captureData(SourceFile:73)
	at com.socure.idplus.devicerisk.androidsdk.viewModel.AllDeviceViewModel.captureData(SourceFile:61)
```

**Comment by buraksocure** [comment]
Hi @Pavel87 Thanks for reporting this. We will review and get back to you. 

CC: @rburunkayaSocure

**Comment by rburunkayaSocure** [comment]
Hi @Pavel87 we need more information to identify the issue. Could you please reach out to us on dse@socure.com email? Thanks

**Comment by buraksocure** [comment]
Resolved with 2.0.4 (https://github.com/socure-inc/socure-sigmadevice-sdk-android/releases/tag/2.0.4)
